modules/topic/index.js

```diff
@@ -0,0 +1,79 @@
+let bot;
+let chunks = null;
+let numChunksAllowed = 16;
+const chunkFileName = 'chunks.json';
+
+module.exports.init = function (b) {
+  bot = b;
+  bot.readDataFile(chunkFileName, (err, jsonData) => {
+    if (err) {
+      bot.log.info('initializing chunks');
+      chunks = {};
+    } else {
+      try {
+        chunks = JSON.parse(jsonData);
+      } catch (ex) {
+        bot.log.warn('Corrupted chunks file! Resetting dict...');
+        chunks = {};
+      }
+    }
+  });
+}
+
+  let store = (chunk, user) => {
+    chunks.chunk = user;
+    bot.writeDataFile(chunkFileName, JSON.stringify(chunks), (err) => {
+      if (err) bot.log.error(`Error writing chunks file: ${err}`);
+    });
+  };
+
+module.exports.commands = {
+  topic(a, parts, b, c, user, channel) {
+    let msg;
+    chan = bot.client.chanData(channel);
+    switch(parts[0]) {
+      case 'help':
+        bot.client.say(channel, helpMsg);
```

Comment:
In general, the `reply` object passed in to the module should be used whenever possible. `bot.client` is not meant to be a stable interface, so we should use it as little as possible.